src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,20 @@
 -->
 
 ## Upcoming Release
+* This release introduces the cmdlets for the preview features of Continuous Backup(Point in time restore):
+  - Introduced support for creating accounts with continuous mode backup policy.
+  - Introduced support for Point in time restore for accounts with continuous mode backup policy.
+  - Introduced support to update the backup interval and backup retention for accounts with periodic mode backup policy.
+  - Introduced support to list the restorable resources in a live database account.
+  - New-AzCosmosBAccount cmdlet is updated with new parameters: IsRestoreReqeust, RestoreSource, RestoreTimestampInUtc, DatabasesToRestore, BackupPolicy.
+  - Update-AzCosmosDBAccount cmdlet is updated with new parameters: BackupPolicy.
+  - The following cmdlets are added:
+    - New-AzCosmosDBRestore, New-AzCosmosDBDatabaseToRestore, Get-AzCosmosDBRestorableDatabaseAccount,
+    - Get-AzCosmosDBSqlRestorableDatabase, Get-AzCosmosDBSqlRestorableContainer, Get-AzCosmosDBSqlRestorableResource,
+    - Get-AzCosmosDBMongoDBRestorableDatabase, Get-AzCosmosDBMongoDBRestorableCollection, Get-AzCosmosDBMongoDBRestorableResource.
+  - Removed the support for throughput Migration. (Please use the stable version of Az.CosmosDB Powershell module if you intend to use this feature).
+  - Removed AnalyticalStorageTTL from SqlContainer datamodel (Please use the stable version of Az.CosmosDB Powershell module if you intend to use this feature).
+* Please note this is a preview version. The cmdlets introduced here may be changed or removed in a future release.
```

Comment:
Hi if you add these cmdlets in stable release, breaking change is only allowed in major release.

Reply:
Should we increase the major version if we add the new cmdlets only? (no breaking change to existing cmdlets)

Reply:
If no breaking to existing cmdlets, we needn't increase major version. But if you announce the cmdlets will change in future, I recommend you to avoid adding them in stable version

Reply:
I have updated the message in update log.